config/rollup.config.js

```diff
@@ -1,8 +1,13 @@
+import fs from 'fs';
 import nodeResolve from 'rollup-plugin-node-resolve';
 import babel from 'rollup-plugin-babel';
 import memory from 'rollup-plugin-memory';
 
+const pkg = JSON.parse(fs.readFileSync('./package.json'));
```

Comment:
@developit Oh, I've seen it [used](https://github.com/rollup/rollup-starter-project/blob/f076671cf11f02408487b9d4bc52458301e49605/rollup.config.js#L5) [elsewhere](https://github.com/rollup/rollup-plugin-babel/blob/8c1bc1c7db6c3acc06755e7a5d411cfb86c3e19c/rollup.config.js#L3) but if it doesn't work here then I'll take your word for it :)